cpp/src/slice2cpp/Gen.cpp

```diff
@@ -635,6 +635,11 @@ Slice::Gen::~Gen()
     H << "\n\n#include <IceUtil/PopDisableWarnings.h>";
     H << "\n#endif\n";
     C << '\n';
+
+    H.close();
+    C.close();
+    implH.close();
```

Comment:
I thought we already removed `--impl`, is this a leftover?

Reply:
Could be! Maybe I can remove it.

Reply:
Removed!